source/reference/minio-cli/minio-mc-admin/mc-admin-speedtest.rst

```diff
@@ -0,0 +1,65 @@
+======================
+``mc admin speedtest``
+======================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc admin speedtest
+
+Description
+-----------
+
+.. start-mc-admin-speedtest-desc
+
+The :mc-cmd:`mc admin speedtest` command tests throughputs per host with PUT and GET operations.
+
+:mc-cmd:`mc admin speedtest` is not supported on standalone MinIO deployments
+or MinIO Gateway deployments and requires :minio-release:`RELEASE.2021-07-30T00-02-00Z` or later.
```

Comment:
```suggestion

:mc-cmd:`mc admin speedtest` requires a distributed MinIO deployment running  :minio-release:`RELEASE.2021-07-30T00-02-00Z` or later. :mc-cmd:`~mc admin speedtest` does not support standalone or MinIO Gateway deployments.

```

Reply:
Minor swapping of order here so we go from 'supported' -> 'nonsupported' in flow.